lib/src/bloc/i_movies_bloc.dart

```diff
@@ -6,5 +6,9 @@ import 'bloc.dart';
 abstract class IMoviesBloc extends Bloc {
   Stream<Movies> get stream;
 
-  void fetchMovies();
+  Stream<Movies> get swiper;
```

Comment:
try to find more descriptive names for this streams accordingly with what they do, and apply that on the movie bloc class, to keep consistency

Reply:
done